syntax = "proto3";
import "VECTOR3.proto";

package Protocol;

message ENTER_SCENE_RES{
	int32 result = 1;
	string sceneName = 2;
	VECTOR3 position = 3;
}